scripts/prompts.ts

```diff
@@ -78,28 +92,44 @@ export async function promptPrepareRelease(opts: BuildOptions): Promise<PrepareR
       type: 'confirm',
       name: 'confirm',
       message: (answers: any) => {
-        const version = answers.version ?? answers.specifiedVersion;
+        const version = determineAnsweredVersionToUse(answers);
         return `Prepare release ${opts.vermoji}  ${color.yellow(version)} from ${oldVersion}. Continue?`;
       },
     },
   ];
 
-  let answers: PrepareReleasePromptAnswers;
   try {
-    answers = await inquirer.prompt<PrepareReleasePromptAnswers>(prompts);
+    const answers = await inquirer.prompt<PrepareReleasePromptAnswers>(prompts);
+    return {
+      confirm: answers.confirm,
+      versionToUse: determineAnsweredVersionToUse(answers),
+    };
   } catch (err: any) {
     console.log('\n', color.red(err), '\n');
     process.exit(0);
   }
-  return answers;
+}
+
+/**
+ * Helper function to determine which version string to use.
+ *
+ * Due to a bug in Inquirer, the version to publish Stencil under needs to be specified under two different fields when
+ * the release preparation scripts diverge in questioning. This function ensures the logic for determining which of
+ * those two answers is to be used.
+ *
+ * @param answers user provided answers to pick from
+ * @returns the version string to use. defaults to 'UNKNOWN' if the version cannot be determined.
+ */
+export function determineAnsweredVersionToUse(answers: PrepareReleasePromptAnswers): string {
+  return answers.version ? answers.version : answers.specifiedVersion ? answers.specifiedVersion : 'UNKNOWN';
```

Comment:
Nit: Nullish coalescing might be a little easier to read:
```suggestion
  return answers.version ?? answers.specifiedVersion ?? 'UNKNOWN';
```

Reply:
We could, although I don't think that necessarily handles the case where either is set as an empty string.